lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Since will need to set rentals from a book instance, I'd advise assigning both getters and setters for the @rentals instance variable.
- [ ] Additionally please make sure to implement a method for adding rentals since a `book` instance will be associated with multiple `rentals`, Here's a link to the [previous lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) you can use as a reference.